src/org/exist/security/UnixStylePermission.java

```diff
@@ -78,7 +76,11 @@ public UnixStylePermission(final SecurityManager sm, final int ownerId, final in
      */
     @Override
     public Account getOwner() {
-        return sm.getAccount(getOwnerId());
+        Account account = sm.getAccount(getOwnerId());
+        if (account == null) {
+            return sm.getSystemSubject();
```

Comment:
is it safe to assume "no subject" -> "system subject" ? Better to have 'guest'may be ?